index.html

```diff
@@ -50,35 +50,79 @@ <h1>My Name</h1>
     <main tabindex="0">
       <section id="about">
         <header><h2>About Me</h2></header>
-        <p>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla nisi
-          excepturi quidem, eum rem doloribus quam aspernatur hic enim eligendi
-          commodi minima ullam necessitatibus, cumque blanditiis, nihil magni
-          amet consectetur?
+        <p class="abt-me">
+          I'm Berwin Eibo, a dedicated software developer with a background in
+          education. My years of experience in teaching have honed my
+          communication skills and taught me the value of teamwork. I'm
+          passionate about creating innovative solutions to real-world problems
+          through code. I thrive on learning and eagerly embrace new challenges
+          in the dynamic world of technology.
         </p>
       </section>
+      <!-- projects -->
       <section id="projects">
         <header><h2>Projects Showcase</h2></header>
-        <p>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla nisi
-          excepturi quidem, eum rem doloribus quam aspernatur hic enim eligendi
-          commodi minima ullam necessitatibus, cumque blanditiis, nihil magni
-          amet consectetur?
-        </p>
+        <div class="projects-container">
+          <div class="project1">
```

Comment:
```suggestion
          <div class="project">
```

1. Should this be a div? Are there other html elements that express this layout more clearly. Is this a list of three items? Are these sections? There are probably a few reasonable answers -- but think about it please!
2. What is the purpose of classes in CSS and HTML? Can you talk through why we use classes, specfically, over ids or element names, or other ways of identifying nodes?